src/parallel_inline.workflows.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START workflows_parallel_inline]
+main:
+  params: []
+  steps:
+    - init:
+        assign:
+          - workflow_id: "translate"
+          - texts_to_translate:
+              - text: "hello world!"
+                source: "en"
+                target: "fr"
+              - text: "你好 世界！"
+                source: "zh-CN"
+                target: "en"
+              - text: "No hablo español!"
+                source: "es"
+                target: "en"
+          - translated: ["", "", ""]
+    - parallel_translate:
+        parallel:
+          shared: [translated]  # to write to this variable, we must share it
```

Comment:
nit: Some files have this comment here and some above on line 32. Can you standardize it to be in one location

Reply:
Yes, ack.